### ISSUE TITLE
Docker deployment mixes migration and running, no entrypoint

### ISSUE THREAD

**Issue by wbl** [blockscout/blockscout#5175]
It seem that the dockerfile doesn't have an entrypoint and will run db deployments when started. This seems a bit unusual compared to separately managing the migrations from running the application.

**Comment by wbl** [comment]
Also having releases of a container corresponding to tagged ones on github would be nice.

**Comment by wbl** [comment]
It doesn't seem that the docker image can actually run against a geth node.

**Comment by vbaranov** [comment]
@wbl It can. Min set of env vars you need to set is:
```
export ETHEREUM_JSONRPC_VARIANT=geth
export ETHEREUM_JSONRPC_HTTP_URL=http://localhost:8545
export ETHEREUM_JSONRPC_TRACE_URL=http://localhost:8545
export DATABASE_URL=postgresql://...
```

**Comment by wbl** [comment]
https://docs.blockscout.com/for-developers/information-and-settings/env-variables is the page I should have been looking at to understand this.

**Comment by vbaranov** [comment]
@wbl I removed mixig of migration and running in Makefile tasks.
Also, docker-compose configs have been created for simplification of Blockscout bootstraphttps://github.com/blockscout/blockscout/blob/master/docker-compose/README.md.
It is available in [v4.1.2](https://github.com/blockscout/blockscout/releases/tag/v4.1.2-beta). Please let me know if you think, that Docker deployment can be furthure improved more. For, now I am closing this issue.
